test/ipcFixtures.hpp

```diff
@@ -93,6 +93,9 @@ struct umfIpcTest : umf_test::test,
         providerParamsDestroy = provider_params_destroy;
         memAccessor = accessor;
         openedIpcCacheSize = getOpenedIpcCacheSize();
+
+        int ncores = utils_get_num_cores();
+        NTHREADS = (ncores > 10) ? ncores : 10;
```

Comment:
could we rename NTHREADS to something like numThreads? why this is named using caps if this isn't a #define or something special?

Reply:
OK, replaced with `numThreads`. I suppose it was in CAPS, because it has been a `static constexpr` so far.
Done

Reply:
Done